<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWireGuard</class>
 <widget class="QWidget" name="EditWireGuard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">EditVMess</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QLineEdit" name="peer_public_key"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="reserved"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="peerPublicKey_l">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>Peer Public Key</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="private_key"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="mtu_l">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string notr="true">MTU</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="reserved_l">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>Reserved</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="local_address"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="preSharedKey_l">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>Pre Shared Key</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="pre_shared_key"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="privateKey_l">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>Private Key</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="localAddress_l">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>Local Address</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSpinBox" name="wireguard_mtu">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="minimum">
      <number>576</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>1408</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
